opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -10,7 +10,17 @@
     | noop_layer_disabled         | 12 ns       |
     | noop_layer_enabled          | 25 ns       |
     | ot_layer_disabled           | 19 ns       |
-    | ot_layer_enabled            | 196 ns      |
+    | ot_layer_enabled            | 167 ns      |
+
+    Hardware: Apple M4 Pro
+    Total Number of Cores:	14 (10 performance and 4 efficiency)
```

Comment:
It'd be great if you could also add the CPU speed as part of the details.

Reply:
I need to find that detail. @lalitb do you know?